src/app/api/session/route.ts

```
@@ -4,10 +4,15 @@ import prisma from '@/lib/prisma';
 import { z } from 'zod';
 import {
   createWorkoutSession,
-  calculateSessionMetrics
+  calculateSessionMetrics,
```

ESLint

[error] 'calculateSessionMetrics' is defined but never used.

(@typescript-eslint/no-unused-vars)